src/legacy/core_plugins/vis_type_vislib/public/plugin.ts

```diff
@@ -39,6 +39,7 @@ import {
   createGoalVisTypeDefinition,
 } from './vis_type_vislib_vis_types';
 import { ChartsPluginSetup } from '../../../../plugins/charts/public';
+import { ConfigShema as VisTypeXyConfigShema } from '../../vis_type_xy';
```

Comment:
Typo `shema` instead of `schema` - got autocompleted to a few other places as well.

Reply:
6847fcd